tools/flowc/flowc_usage.flow

```diff
@@ -144,9 +145,10 @@ printUsage(config) {
 	prih(help,"      outfolder=<FOLDER>        Save the typed result of all test cases in the given folder");
 	prih(help,"      verify-types=0            Do not verify the assigned types after typechecking is done");
 	prih(help,"");
-	prih(help,"      server-mode=1             Run flowc in a server mode");
```

Comment:
Is the VS code plugin updated accordingly? Why rename this?

Reply:
Yes, flow vscode plugin is updated, a new version is compiled. Users need to re-install it.